packages/docsify-server-renderer/index.js

```diff
@@ -95,9 +112,8 @@ export default class Renderer {
       this._renderHtml('cover', await this._render(coverFile), 'cover');
     }
 
-    const html = this.html;
+    const html = this.isRemoteUrl ? DOMPurify.sanitize(this.html) : this.html;
```

Comment:
This `docsify-server-renderer` package is only for server side though, right? So the bundle size may actually not matter much in this case.

Then this won't fix the client-side issue mentioned in https://github.com/docsifyjs/docsify/issues/1126.

For that, I think we need to modify something in https://github.com/docsifyjs/docsify/tree/develop/src/core/router.

Reply:
Oh wait, I see the fix for client side too. Let me update that.